guides/common/modules/proc_configuring-simplified-alternate-content-sources.adoc

```diff
@@ -0,0 +1,41 @@
+[id="Configuring_Simplified_Alternate_Content_Sources_{context}"]
+= Configuring Simplified Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *Simplified*.
+. Select the *Content type*.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the *Description* field, provide a description for the ACS.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Optional: Select `Use HTTP proxies` if you want the ACS to use the {SmartProxy}'s HTTP proxy.
```

Comment:
```suggestion
. Optional: Select *Use HTTP proxies* if you want the ACS to use the {SmartProxy}'s HTTP proxy.
```

Please do not use `{SmartProxy}s`. Maybe `{SmartProxyServer}s`? See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L109